OPERATING.md

```diff
@@ -28,7 +28,16 @@ yyyy-MM-dd HH:mm:ss.SSS Level=INFO Logger=u.n.a.g.e.m.AgentDirectoryExtractor Co
 
 ## Object Storage (AWS S3 / Azure Blob)
 
-TODO: Document storage scheme
+- Data stored:
+  - The patients transformed structured record once it has been transformed into an ehrExtract that is ready to send to MHS Outbound
+  - Attachments associated with the patients structured record once they have been transformed into an outbound message for sending to MHS Outbound
+- Filename convention:
+  - EhrExtracts files are named as a concatonation of {conversationId}/{conversationId}.json
```

Comment:
is it correct that we use the same conversationId twice as the file names?

Reply:
Yes, for the transformed structured record